pact/sale-contracts/dutch-auction/dutch-auction.repl

```diff
@@ -279,9 +285,21 @@
     (get-current-price (read-string 'sale-id))
   )
 
-  (env-chain-data {"block-time": (time "2023-10-07T23:45:00Z")})
-  (expect "Just before closing the price is at reserve price"
-    50.00
+  (env-chain-data {"block-time": (time "2023-10-07T23:59:59Z")})
+  (expect "Price keeps dropping close to reserve price"
+    50.35
+    (get-current-price (read-string 'sale-id))
+  )
+
+  (env-chain-data {"block-time": (time "2023-10-08T00:00:00Z")})
+  (expect "At closing the price drops to reserve price"
```

Comment:
The downside of this is that you'll never be able to buy at the reserve price since it's only valid for one second.